src/toniarts/openkeeper/game/state/MainMenuState.java

```diff
@@ -103,6 +108,8 @@
     private KwdFile kwdFile;
     private final MouseEventListener mouseListener = new MouseEventListener(this);
     private Vector3f startLocation;
+    private static final Logger logger = Logger.getLogger(MainMenuState.class.getName());
+    HashMap<String, String[]> cutscenes = new HashMap<>();

```

Comment:
Looks great! Only thing I can nitpick is that this could be a static final and instanced in static {} instead of initialize() :)